README.md

```diff
@@ -39,7 +39,7 @@ When the reader has completed this Code Pattern, they will understand how to:
 1. [Clone the repository](#1-clone-the-repository)
 1. [Create a Databases for PostgreSQL DB](#2-create-a-databases-for-postgresql-db)
```

Comment:
this line should have been changed, not the one 2 lines below.

Reply:
Thanks. Fixed.